hane-syntax/src/lower.rs

```diff
@@ -63,6 +80,60 @@ impl Command {
                 let ttype = ttype.lower(global, &mut names)?;
                 global.insert(name.clone());
                 LoweredCommandVariant::Axiom(name, ttype)
+            },
+            CommandVariant::Inductive(mut bodies) => {
```

Comment:
It would be nice with overall comments describing why you are doing what you are doing

Reply:
I think I've done that